distribution/docs/src/main/resources/_contents/_sources/solrcloud-catalog-provider-contents.adoc

```diff
@@ -0,0 +1,85 @@
+==== Solr Cloud Catalog Provider
+
+Solr Cloud is a cluster of Solr Server instances that support fault tolerance and high availability.
+Each Solr Server instance in Solr Cloud provides a Solr Admin GUI.
+
+Synchronization between Solr Server instances is managed by Zookeeper.
+Zookeeper routes client requests to the proper Solr Server instance in the the Solr Cloud cluster.
+
+===== When to Use the Solr Cloud Catalog Provider
+
+If the ${branding} needs to be accessible for a higher than normal period with low instances of downtime, then Solr Cloud should be used.
+
+===== High Availability (Beta)
```

Comment:
Thinking I might just get rid of this section and move the note about this being in beta right above "When to Use the Solr Cloud Catalog Provider". What do you think?